api/Makefile

```diff
@@ -46,7 +46,8 @@ gen: clean            ## Generate files.
 		swagger/header.json \
 		serverpb/json/serverpb.json \
 		inventorypb/json/inventorypb.json \
-		managementpb/json/managementpb.json
+		managementpb/json/managementpb.json \
+		managementpb/dbaas/json/dbaas.json
```

Comment:
No, that line should not be there – the comment above methods that

Reply:
9fb795e5678bb5edb3a4c0d126592563c2a9edac

Reply:
Why dbaas is private?

Reply:
> the comment above mentions that

That comments reads (in my branch):

> 	# generate public API spec, omit agentlocalpb (always private),
>	# and backups, dbaas, and qanpb (not v1 yet)

Reply:
I see that it's mentioned, but why?

Reply:
Why it is not v1 stable yet? Because it is not stable yet – we reserve the right to make breaking changes. Think as version 0 in SemVer